xbmc/input/WindowTranslator.h

```diff
@@ -52,5 +52,26 @@ class CWindowTranslator
   static int GetFallbackWindow(int windowId);
 
 private:
-  static std::map<int, const char*> CreateReverseWindowMapping();
+  struct WindowMapItem
+  {
+    const char *windowName;
+    int windowId;
+  };
+
+  struct WindowNameCompare
+  {
+    bool operator()(const WindowMapItem &lhs, const WindowMapItem &rhs) const;
+  };
+
+  struct WindowIDCompare
+  {
+    bool operator()(const WindowMapItem &lhs, const WindowMapItem &rhs) const;
+  };
+
+  using WindowMapByName = std::set<WindowMapItem, WindowNameCompare>;
+  using WindowMapByID = std::set<WindowMapItem, WindowIDCompare>;
+
+  static WindowMapByID CreateReverseWindowMapping();
+
+  static const WindowMapByName WindowMapping;
```

Comment:
Minor, but could we adjust the name to `WindowMappingByName` and `CreateWindowMappingByID`